src/tof/model.py

```diff
@@ -31,18 +51,71 @@ class Model:
 
     def __init__(
         self,
-        choppers: Union[Chopper, List[Chopper]],
-        detectors: Union[Detector, List[Detector]],
         pulse: Pulse,
+        choppers: Optional[
+            Union[Chopper, List[Chopper], Tuple[Chopper, ...], Dict[str, Chopper]]
```

Comment:
Seems odd to accept both `Iterable[Chopper]` (list, tuple) and `Dict`?

Reply:
It's actually a good point, because if the user provides a dict where the keys do not agree with the `chopper.name`, then it may mess things up elsewhere. So I will remove the option of giving a dict. Thanks